web-components/src/[sandbox]/examples/table.ts

```diff
@@ -8,11 +8,14 @@
 
 import "@/components/table/Table";
 import { html } from "lit-element";
+import { tableWarns } from "@/[sandbox]/sandbox.mock";
+
+const tableError = [6, 27];
```

Comment:
so this is like a cell index? sorry Lena but this does not make sense to me. By a coordinate i meant: Column 3, cell 3 ([3,3]). so if you would like to pass 2 errors, they will be [[2,3], [9,3]], not [6, 27]

Reply:
Updated